contracts/BosonRouter.sol

```diff
@@ -1217,16 +1253,17 @@ contract BosonRouter is
      * @param _condition condition that will be checked when a user commits using a conditional token
      * @param _registerConditionalCommit indicates whether Gate.registerVoucherSetId should be called. Gate.registerVoucherSetId can also be called separately
      */
-    function finalizeConditionalOrder(uint256 _tokenIdSupply, address _gateAddress, uint256 _conditionalTokenId, Condition _condition, bool _registerConditionalCommit) internal {
+    function finalizeConditionalOrder(uint256 _tokenIdSupply, address _gateAddress, uint256 _conditionalTokenId, Condition _condition, uint256 _threshold, bool _registerConditionalCommit) internal {
```

Comment:
Since `finalizeConditionalOrder` actually takes in all elements of a struct `ConditionalCommitInfo`, it might be sensible that struck as a whole is passed to this function.

`finalizeConditionalOrder(uint256 _tokenIdSupply, ConditionalCommitInfo calldata _conditionalCommitInfo)`

This saves a bit of a code. You then also don't have do destruct `_conditionalCommitInfo` every time you call `finalizeConditionalOrder` from other functions.

Reply:
Agreed. I thought of this after I pushed the PR. I made this change and will include it in the next commit.